web/api/v1/api.go

```diff
@@ -242,6 +273,8 @@ func (api *API) Register(r *route.Router) {
 	r.Get("/alertmanagers", wrap(api.alertmanagers))
 
 	r.Get("/status/config", wrap(api.serveConfig))
+	r.Get("/runtimeinfo", wrap(api.serveRuntimeInfo))
+	r.Get("/buildinfo", wrap(api.serveBuildInfo))
```

Comment:
Probably these would also be:

- `/status/runtime`
- `/status/buildinfo`

Reply:
I decided to omit the "status" part since alertmanager route doesn't use it. Should I change the alertmanagers path as well?

Reply:
We can't break the existing v1 API, but I guess `/api/v1/alertmanagers` is also far-enough removed from the main status of the Prometheus server (like `/api/v1/targets`, `/api/v1/service-discovery`, and the like) that I wouldn't really expect it to be under `/api/v1/status/...`. I guess someone still decided to put it onto the `/status` page because they didn't feel creating an extra web page for that little information was worth it :man_shrugging: 

But runtime and build info are infos that are closely tied to the server itself, so I think they will best live under `/api/status/...`